spec/factories/infrastructures.rb

```diff
@@ -50,6 +50,46 @@
                                     "timeout": ""
                                   }
                                 }
+                              },
+                              {
+                                "type": "elasticsearch",
+                                "desired_num_replicas": 1,
+                                "min_available_replicas": 0,
+                                "definition": {
+                                  "container_type": "stateless",
+                                  "strategy": "RollingUpdate",
+                                  "allow_failure": "false",
+                                  "source": {
+                                    "mode": "pull",              # can be local or pull. default is pull.
+                                    "alias": "lxd-ubuntu-minimal-elasticsearch-1.1.0-8",
+                                    "remote": {
+                                      "name": "barito-registry"
+                                    },
+                                    "fingerprint": "",
+                                    "source_type": "image"
+                                  },
+                                  "resource": {
+                                    "cpu_limit": "0-2",
```

Comment:
have we implemented this when creating new container?
and what is meaning of "0-2"?

Reply:
haven't implemented this feature yet, Mas.

On LXC you can have exact value or range value.
https://www.jamescoyle.net/how-to/2532-setting-cpu-resource-limits-with-lxc

Reply:
So based on value, it will always use physical CPU core 0, 1, 2 on the host? i think its better to change dynamically value for next